site/includes/deptfeatures-wet-126.hbs

```diff
@@ -0,0 +1,23 @@
+<div class="gc-prtts mrgn-tp-md">
+	<div class="container">
+		<div class="row">
+			<section class="provisonal col-md-8 mrgn-bttm-md mrgn-tp-lg">
+				<h2 class="wb-inv">Feature</h2>
+				<a href="#">
+					<figure class="row">
+						<figcaption class="pull-right col-md-6 col-xs-12">{{{i18n "de-title-link"}}}</figcaption>
+						<div class="pull-left col-md-6 col-xs-12">
+							<img src="{{assets}}/img/360x203.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm"/>
+						</div>
+						<div class="pull-right col-md-6 col-xs-12">
+							<p>{{{i18n "de-description"}}}</p>
+						</div>
+					</figure>
+				</a>
+			</section>
+			<section class="col-md-4 mrgn-bttm-sm">
```

Comment:
Change this for a `div` and keep the `section` element with it's top heading

Reply:
This has been done.